tee-worker/ts-tests/examples/direct-invocation/util.ts

```diff
@@ -260,5 +294,15 @@ export const createRequest = async (
     } else {
         cyphertext = compactAddLength(bufferToU8a(encryptWithTeeShieldingKey(teeShieldingKey, top)));
     }
+
     return parachain_api.createType('Request', { shard: hexToU8a(mrenclave), cyphertext }).toU8a();
 };
+
+// decode nonce
+export function decodeNonce (nonceInHex: any) {
```

Comment:
I think you are expecting `string` here, can we use strict type ?

Reply:
yes